lib/RefBackend/RefBackend.cpp

```diff
@@ -64,6 +64,30 @@ static Type getAbiTypeForMemRef(Type type) {
   return UnrankedMemRefType::get(type.cast<MemRefType>().getElementType(), 0);
 }
 
+// Passes the return op operands `val` to `funOp`. Also, adds the op to the
+// `toErase` vector.
+static void replaceCallToFunction(OpBuilder b, ReturnOp op, FuncOp funcOp,
+                                  Value val,
+                                  SmallVectorImpl<Operation *> &toErase) {
+  b.create<mlir::CallOp>(op.getLoc(), funcOp, val);
+  b.create<mlir::ReturnOp>(op.getLoc());
+  toErase.push_back(op);
+}
+
+// Checks whether the return op is compatible and the respective calling
```

Comment:
nit: I would be a bit more descriptive as to what the return op is compatible with. Maybe you can replace `compatible` with `munge-compatible`.

Reply:
Done, thanks.